src/ip_grid_descriptor_mod.f90

```diff
@@ -1,3 +1,17 @@
+!> @file
+!! @brief Uses derived type grid descriptor objects to abstract away
+!! the raw Grib-1 and Grib-2 grid definitions.
+!! @author Kyle Gerheiser
+
+!> @brief Uses derived type grid descriptor objects to abstract away
+!!the raw Grib-1 and Grib-2 grid definitions.
+!! @detail
```

Comment:
I dont think we need or want the @detail tag. Instead, just leave an empty line to get a new paragraph. So:

```
!> @brief Uses derived type grid descriptor objects to abstract away
!! the raw Grib-1 and Grib-2 grid definitions.
!! 
!! An abstract ip_grid_descriptor class is subclassed by the
...
```